spec/ss_sample_blast_no_hits.asn

```diff
@@ -0,0 +1,130 @@
+Blast4-archive ::= {
```

Comment:
Archive format (11) is binary, I believe. How come using text file for test suite?